mainwindow.cpp

```diff
@@ -322,3 +346,103 @@ void MainWindow::selectNoStreams() {
 		item->setCheckState(Qt::Unchecked);
 	}
 }
+
+void MainWindow::buildFilename() {
+	// This function is only called when a widget within Location Builder is activated.
+
+	// Fill root folder with default if needed.
+	if (ui->rootEdit->text().isEmpty()) {
+			QStringList fileparts =
+				QStringList(QStandardPaths::writableLocation(QStandardPaths::DocumentsLocation))
```

Comment:
Nice find! We should use also this as default path in case the config file doesn't contain one.